pipelines/A4/A4.R

```diff
@@ -0,0 +1,57 @@
+# -------------------------------------------------------------------------
+# Step: Get the data
+# input: google sheet
+# output: ds$data, ds$dir
+# -------------------------------------------------------------------------
+
+ds <- create_dataset(id = "A4")
+ds <- download_data(ds)
+ds$data
+
+# -------------------------------------------------------------------------
+# Step: Clean the data and add spatial unit
+#   input: ds$data
+#   output: ds$postgres_export: spatial unit uid added
+# -------------------------------------------------------------------------
+
+ds$postgres_export <- ds$data %>%
+  janitor::clean_names() %>%
+  tibble::as_tibble() %>%
+  dplyr::filter(
+    !stringr::str_starts(rubrik, "Energieumwandlung")
+  ) %>%
+  dplyr::mutate(
+    spatialunit_uid = spatial_mapping_country()
+  ) %>%
+  dplyr::rename(
+    "verbrauchsart" = rubrik,
+    "anzahl" = tj
+  ) %>% tidyr::pivot_wider(
+    names_from = c("verbrauchsart"),
+    values_from = anzahl,
+    names_prefix = "terajoule_"
+  ) %>%
+  janitor::clean_names()
+colnames(ds$export)
```

Comment:
```suggestion
colnames(ds$postgres_export)
```
(or maybe the line can be removed)